### PR TITLE
Add streaming playlist to Media app

### PR DESCRIPTION
**Pull request by jovirkku**

### CODE REVIEW

File: Support/Help_Articles/Media/README.md [hunk 1]
```diff
@@ -44,11 +44,51 @@ From the All songs view songs can be added to playlists, added to play queue or
...
 
-## Related articles
+# Adding a playlist for Internet streaming
+
+The Media app supports Internet streaming (e.g. Internet radio stations) but there is no support for this in the user-interface of this app. 
+
+This chapter has the instructions for adding a playlist for Internet streaming. The playlist makes it easy to use the streaming as the list appears among the song playlists and can be used in the same way.See the two pictures below.
+
+Making the streaming playlist requires that some lines of text are written to a new playlist file in directory ```/home/defaultuser/Music/playlists/```. We have given the name "SomeRadios.pls" for the file but feel free to change it (the extension must remain as pls). You will need to enable the **[Developer mode](/Support/Help_Articles/Enabling_Developer_Mode/)** on your phone.
+Get the root right and create/edit the file like this:
```
Comment (vigejolla) [inline Support/Help_Articles/Media/README.md:58]: "Get the root right"? Maybe "Get the root permissions"?

File: Support/Help_Articles/Media/README.md [hunk 2]
```diff
@@ -44,11 +44,51 @@ From the All songs view songs can be added to playlists, added to play queue or
...
+
+```
+devel-su
+vi /home/defaultuser/Music/playlists/SomeRadios.pls
+```
+
+Our example file below has three examples (a random selection) of radio stations streaming music. The file must have the tag "[playlist]" on the first line and "NumberOfEntries=" with the correct digit on the last line.
```
Comment (vigejolla) [inline Support/Help_Articles/Media/README.md:65]: Using the word "tag" for describing the thingie which defines the section name felt a bit wierd, so I did a little research to find out what it should actually be called. Turns out both wikipedia and microsoft documentation are careful not to call it anything! So, let's just call it a tag then :)

Comment (vigejolla) [reply Support/Help_Articles/Media/README.md:65]:
```suggestion
Our example file below has three examples (a random selection) of radio stations streaming music. The file must have the tag `[playlist]` on the first line and `NumberOfEntries=` with the correct digit on the last line. 
```

**Review by vigejolla — DISMISSED** [review]
just some nitpicks

File: Support/Help_Articles/Media/README.md (hunk 2 above)
Comment (jovirkku) [reply Support/Help_Articles/Media/README.md]: All comments handled.

**Review by rainemak — APPROVED** [review]

**Pull request merged**